content/creating-addons/react/en/add-to-catalog.md

```diff
@@ -0,0 +1,118 @@
+
+## Catalog Metadata
+
+Along with the module related info you also need to specify some metadata for the Storybook Addon Catalog.
+
+![catalog metadata includes tags, compatibility, authors, etc.](../../images/catalog-metadata.png)
+
+Some of this info is pre-configured by the Addon Kit. Things like a display name, icon or frameworks compatibility is specified via the storybook property. For the complete specification of the metadata API check out the [Addon metadata documentation](https://storybook.js.org/docs/react/addons/addon-catalog/#addon-metadata).
+
+```json:title=package.json
+{
+  ...
+  "name": "my-storybook-addon",
+  "version": "1.0.0",
+  "description": "My first storybook addon",
+  "author": "Your Name",
+  "storybook": {
+    "displayName": "My Storybook Addon",
+    "unsupportedFrameworks": ["react-native"],
+    "icon": "https://yoursite.com/link-to-your-icon.png"
+  },
+  "keywords": ["storybook-addons", "appearance", "style", "css", "layout", "debug"]
+  ...
+}
+```
+
+The keywords property here maps to catalog tags. For example, the storybook-addons tag ensures that your addon will get into the catalog. And appearance is a top-level category. The rest help with searchability of your addon.
+
+## Publishing to NPM
+
+The last step is to actually publish the addon. The Addon Kit comes pre-configured with [Auto](https://github.com/intuit/auto) for release management. It generates a changelog and pushes it to both GitHub and npm. Therefore, you need to configure access to both.
+
+1. Authenticate using [npm adduser](https://docs.npmjs.com/cli/adduser.html)
+2. Create an [access token](https://docs.npmjs.com/creating-and-viewing-access-tokens#creating-access-tokens). You’ll need a token with both _Read and Publish_ permissions.
+3. Similarly, generate a [Github token](https://github.com/settings/tokens). This token will need the repo scope.
+4. Create a `.env` file at the root of your project and add both these tokens to it:
+
+```bash
+GH_TOKEN=value_you_just_got_from_github
+NPM_TOKEN=value_you_just_got_from_npm
+```
+
+Next, **create labels on GitHub**. You’ll use these labels in the future when making changes to the package.
+
+```bash
+npx auto create-labels
+```
+
+If you check on GitHub, you’ll now see a set of labels that Auto would like you to use. Use these to tag future pull requests.
+
+Finally, creating a release
+
+```bash
+npm run release
+```
+
+Which will:
+
+- Build and package the addon code
+- Bump the version
+- Push a release to GitHub and npm
+- Push a changelog to GitHub
+
+There you have it! We’ve successfully published our package to npm and released our first Storybook Addon. There may be a delay in your addon showing up in the catalog because it crawls npm. If your addon isn’t showing up, please file an issue on the catalog repo.
+
+Addons help the community customize and automate Storybook. Publish your addon to share it with thousands of devs. I’m excited to see what you build!
```

Comment:
Add a conclusion.md: 
- Concludes the guide
- **Learn more** section that links to various addon resources, the Outline source code, and the demo Storybook to see it in action
- **Sign up to SB mailing list** embed